README.md

```diff
@@ -89,26 +89,34 @@ docker-compose up
 
 laa-maat-court-data-api application will be running on http://localhost:8090 
 
+
 ### Cloud Platform Set Up 
 
 MAAT API speaks to AWS SQS which exists on cloud platform. It is advisable to have the cloud platform set up locally. 
 
 Follow this link to on board yourself with the LAA cloud platform environment. - https://user-guide.cloud-platform.service.justice.gov.uk/documentation/getting-started/kubectl-config.html#how-to-use-kubectl-to-connect-to-the-cluster
 
-Once you are on board with Cloud Platform, Run the following Command.  
-
+Once you are on board with Cloud Platform, run the following command to display the list of secrets for the messaging queues:
 
 ```sh
-kubectl -n laa-court-data-adaptor-dev get secrets ca-messaging-queues-output -o yaml
+kubectl -n laa-court-data-adaptor-dev get secrets cda-messaging-queues-output -o yaml
 ```
 
-You'll need to decode the secrets using the following command. 
+In order to see their actual values you'll need to decode each secret using the following command:
 
 ```sh
-echo 'Your Secret' | base64 --decode
+echo "<encoded-secret-value>" | base64 --decode
```

Comment:
Another one to add here for anyone who has `jq` installed in their terminal is:
`kubectl -n laa-court-data-adaptor-dev get secret cda-messaging-queues-output -o json | jq '.data|map_values(@base64d)'`
This will retrieve and decode at the same time. :)

Reply:
Looks like a handy utility! Ive added the above section to the readme for those that have it or want to make use of jq.